docs/modules/operation/pages/quick-start/inventory.adoc

```diff
@@ -3,53 +3,50 @@
-* Create policies to manage provisioning behavior.
+== SNMP in {page-component-title}
 
-Refer to xref:operation:deep-dive/provisioning/introduction.adoc[Provisioning] in the Deep Dive section for details about these and other provisioning-related tasks.
+SNMP is used in {page-component-title} to understand network and node topology, collect performance data, and enable SNMP service monitoring of nodes.
+SNMP service monitoring in particular ensures that SNMP is available on a given node; it checks a configurable OID value to understand the status of a particular part, function, or other aspect of the monitored device.
+Tracking the state of this value can be used to trigger a notification if it represents an error or another condition that may need attention.
+
+The service also gathers basic information about a monitored device, including the device type, its name, its location, and so on.
+For more information, see xref:reference:daemons/daemon-config-files/enlinkd.adoc[Enlinkd] in the Reference section.
 
 [[provision-snmp-configuration]]
 == Configure SNMP for provisioning
 
 To collect data through SNMP or monitor the availability of SNMP services on a node, you need to configure SNMP for provisioning.
 This ensures that {page-component-title} can immediately scan newly discovered devices for entities.
-
-Proper SNMP configuration lets {page-component-title} understand network and node topology and automatically enables performance data collection.
-{page-component-title} updates network topology as it provisions nodes.
+{page-component-title} updates the network topology--the visual representation of the monitored network and its components--as it provisions nodes.
 
 This procedure illustrates a very basic SNMP configuration using the default parameters.
-You can customize these values for your own network needs.
+You can customize these values for your own network needs, although the default settings are sufficient for a basic {page-component-title} setup:
 
-. Click the *Gear* symbol in the top-right of the screen.
+. Click the *gear* symbol in the top-right of the screen.
 . Under Provisioning, click *Configure SNMP Community Names by IP Address*.
-. Under General Parameters, specify an IP range using the *First IP Address* and *Last IP Address* boxes, and accept the default values for the other fields.
-. Scroll to the bottom and click *Save Config*.
+. Configure the community names as appropriate for your environment:
+** Enter a valid IP address in the *IP Address* box, and click *Look Up*.
+** Choose an SNMP version (`v2c` or `v1`) from the *Version* list.
+** (Optional) Define a range of IP addresses by typing the first and last addresses of the range into the *First IP Address* and *Last IP Address* boxes, respectively.
+** In the v1/v2c Specific Parameters section, enter the new community string in the *Read Community String* box.
```

Comment:
Is there anything they need to set for Write Community string? Including the Read and Write was something Mike had requested.

Reply:
I think this would also be good to clarify.  I don't believe we write anything by default.  Additionally, this section mentions that SNMPv3 is beyond the scope and I'm not sure it should be.  I think it would be good to cover it here as it's not an uncommon thing someone would want help with getting started on.

Reply:
Thanks! We have a task to update the SNMP documentation as it is, so this will be covered in that ticket (NMS-12873).